app/src/components/posts/VideoPlayer.vue

```diff
@@ -51,127 +78,74 @@ onMounted(() => {
+
+    // Get player user active states
+    player.on(["useractive", "userinactive"], playerUserActiveEventHandler);
+});
+
+onUnmounted(() => {
+    player?.off(["mousemove", "click"], autoHidePlayerControls);
+    player?.off("play", playerPlayEventHandler);
+    player?.off(["useractive", "userinactive"], playerUserActiveEventHandler);
+});
+
+// Set player audio only mode
+watch(audioMode, (newValue) => {
+    player?.audioOnlyMode(newValue);
+    playerUserActiveEventHandler();
 });
 </script>
 
 <style>
 @import "video.js/dist/video-js.min.css";
 @import "videojs-mobile-ui/dist/videojs-mobile-ui.css";
 
-.vjs-tech {
-    @apply !rounded-lg;
-}
-
-.vjs-poster img {
-    @apply !rounded-lg object-cover;
-}
-
-.vjs-big-play-button {
-    @apply !border-none !text-3xl;
-    background-color: rgba(0, 0, 0, 0.3) !important;
-}
-
-.vjs-big-play-button span {
-    @apply !text-5xl;
-}
-
-.vjs-control-bar {
-    @apply !relative !h-full !w-full rounded-md;
-    background-color: rgba(0, 0, 0, 0.3) !important;
-}
-
-:is(.vjs-has-started) .vjs-control-bar {
-    @apply block;
-}
-
-.vjs-control {
-    @apply !float-left !h-11 !w-11 !text-sm !outline-none;
-}
-
-.vjs-picture-in-picture-control {
-    @apply !float-right;
-}
-
-.vjs-play-control {
-    @apply !absolute !inset-0 !m-auto !h-16 !w-16 !text-xl;
-}
-
-.vjs-progress-control {
-    @apply !absolute !bottom-0 !left-0 !pl-2 !pr-0;
-    width: calc(100% - 40px) !important;
-}
-
-.vjs-live-control {
-    @apply !absolute !bottom-0 !left-0 !pl-2 !pr-0;
-}
-
-.vjs-live-display {
-    @apply p-3 pl-2;
-}
-
-.vjs-slider {
-    @apply !rounded-full;
-}
-
-.vjs-slider-bar {
-    @apply !rounded-full;
-}
-
-.vjs-fullscreen-control {
-    @apply !absolute !bottom-0 !right-0;
-}
-
-.vjs-menu {
-    @apply !left-11 !top-0 !outline-none;
-}
-
-.vjs-menu-content {
-    @apply !top-4 !h-fit rounded-md !bg-zinc-50  shadow-lg dark:!bg-zinc-600 dark:text-zinc-100;
-}
-
-.vjs-selected {
-    @apply !bg-zinc-300 font-bold focus:!bg-zinc-300 dark:!bg-zinc-500 focus:dark:!bg-zinc-500;
-}
-
-.vjs-menu-item-text {
-    @apply text-zinc-900 dark:text-zinc-100;
-}
-
-.video-js :not(.vjs-selected) {
-    @apply focus:!bg-transparent;
-}
-
-.vjs-menu-button {
-    @apply !outline-none;
-}
-
-.vjs-menu-item {
-    @apply !p-3 !text-sm !outline-none;
-    width: 100% !important;
-}
+@import "videoPlayerVideoMode.css";
+@import "videoPlayerAudioMode.css";
 </style>
 
 <template>
-    <div class="aspect-video">
-        <video
-            ref="playerElement"
-            class="video-js h-full w-full rounded-lg shadow-md"
-            controls
-            preload="auto"
-            data-setup="{}"
-        ></video>
+    <div class="relative mb-2 rounded-lg bg-zinc-100 shadow-md dark:bg-zinc-800">
+        <div class="relative">
+            <img
+                v-if="audioMode"
+                :src="props.contentParent.image"
+                class="mb-2 aspect-video w-full rounded-lg object-cover"
```

Comment:
Tiny UI improvement: I think it would look a little better if the bottom of the image isn't rounded for the audio player:
![image](https://github.com/bccsa/luminary/assets/39056299/2d6815c5-8077-4a59-ae89-6817992fc4e1)
vs

![image](https://github.com/bccsa/luminary/assets/39056299/514c9d56-e4bd-40d2-99ba-603d9e2da579)

Really like the styling of the player otherwise!